src/modules/python/pbs_hooks/PBS_alps_inventory_check.PY

```diff
@@ -155,11 +158,12 @@ def get_apstat_nids(msg):
     # not ship the subprocess module
     apstat_out = os.popen(cmd_apstat)
 
+    pattern = re.compile("(.*?)\s+(.*?)\s+(.*?)\s+(.*?)\s+(.*)")
+

```

Comment:
@riyazhakki : Did you try regex like this  "(?P<name1>.+?)\s+(?P<name2>.+?)\s+(?P<name3>.+?)\s+(?P<name4>.+?)\s+(.*)"? 
This will not add space to group.